02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+        "guru_cards": [
+            "How is \"spouse\" defined for food stamp applications?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 13,
+        "derived_questions": [
+            "What is the situation between the husband and wife?",
+            "Can they still be considered separated if they live in the same house?",
+            "Do they have any communication with each other?",
+            "Do they share any food or resources?",
+            "Is the wife able to apply for something on her own?"
+        ],
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.5970122321883444,
+                "derived_questions": [
+                    "Is the wife able to apply for something on her own?",
+                    "Can they still be considered separated if they live in the same house?"
+                ],
+                "entire_card": "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?\nAnswer:\nYes, if the adults with whom they are living do not exercise parental control or offer financial support. Clients who are officially emancipated usually fall under this category. If the client not entirely certain or it's close but messy we can error on the side of applying and inform the client the county will help them figure out the household if need be.\nExample: Ms. G is 17 years old. She lives with her aunt. Ms. G works and purchases and prepares her own meals. When questioned about household composition she stated that her aunt does not exercise parental control over her. The case manager certifies Ms. G as a one-person household.\nFood stamps only",
+                "summary": "Children under 18 years of age can be certified as a separate household if the adults they live with do not exercise parental control or offer financial support, and if they are officially emancipated."
+            },
+            "How is \"spouse\" defined for food stamp applications?": {
+                "score_sum": 0.5276235971132459,
+                "derived_questions": [
+                    "What is the situation between the husband and wife?",
+                    "Is the wife able to apply for something on her own?"
+                ],
+                "entire_card": "How is \"spouse\" defined for food stamp applications?\nSpouses are either of two individuals who:\nAre married to each other under applicable state law; or\nAre living together and are holding themselves out to the community as a married couple",
+                "summary": "The husband and wife are living separately in the same house and do not consider themselves married, so the wife may be able to apply for food stamps on her own."
```

Comment:
@aliciaben Review here

Reply:
This summary makes no sense to me "the husband and wife are living separately in the same house and do not consider themselves married" Why is the summary stating that the couple does not consider themselves married - given that the only prompt provided is, "Is the wife able to apply for something on her own?"

Rating: Relevance to the question
🟡 3 - Partially relevant

Rating: Correctness as a summary of the Guru card
🟠 2 - Mixed (partially correct, partially incorrect)

Rating: Helpfulness

🟡 3 - Neutral -

Reply:
> Why is the summary stating that the couple does not consider themselves married - given that the only prompt provided is, "Is the wife able to apply for something on her own?"

@aliciaben The original `question` (line 175 above) is "Husband and wife consider themselves separated, but they still live in the same house. The husband lives on the 2nd floor and the wife lives on the first. They do not speak or share any food. Can the wife apply on her own?"

"Is the wife able to apply for something on her own?" is one of the `derived_questions` that the LLM generated to break down the original `question`. 

For these reviews, we should compare the `summary` to the original `question` (and the `entire_card`).

Reply:
Oh wait, I see what you're saying though -- the `summary` in this case isn't really summarizing the guru card, it's actually generating an answer to the `question`, which is not really what we want it to do